Game.h

```diff
@@ -0,0 +1,122 @@
+#ifndef SNAKE_1_GAME_H
+#define SNAKE_1_GAME_H
+
+#include <iostream>
+
+#include "Board.h"
+#include "Snake.h"
+#include "Food.h"
+
+class Game
+{
+public:
+    Game() { startGame(); }
```

Comment:
i'd make startGame public and call it explicitly in main.cpp. Imo it distinguishes initialization and game logic
But it's your choice